Sources/Caching/DeviceCache.swift

```diff
@@ -547,7 +570,7 @@ fileprivate extension UserDefaults {
             return nil
         }
 
-        return try? JSONDecoder.default.decode(jsonData: data)
+        return try? JSONDecoder.default.decode(jsonData: data, logErrors: true)
```

Comment:
This is just to be explicit, so reader doesn't wonder why we're ignoring errors with `try?`